application/libraries/Yii/Application/ErrorHandler.php

```diff
@@ -0,0 +1,223 @@
+<?php
+
+namespace LimeSurvey\Yii\Application;
+
+use Yii;
+use CWebApplication;
+use CHttpException;
+use LSUserException;
+
+/**
+ * Trait for Application error handling
+ */
+class ErrorHandler
```

Comment:
We should probably note somewhere that this class is NOT related to the CErrorHandler component built in in Yii, https://www.yiiframework.com/doc/api/1.1/CErrorHandler

Reply:
I renamed the class to AppErrorHandler. I think this and the namespace makes it clear that this is not Yii code but LimeSurvey code relating to Yii.